docs/DeveloperGuide.md

```diff
@@ -266,28 +266,51 @@ _{Explain here how the data archiving feature will be implemented}_
-| `* *`    | user                                       | hide private contact details | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name         | locate a person easily                                                 |
-
-_{More to be added}_
+| Priority | As a …           | I want to …                                                                 | So that I can …                                                        |
+|----------|------------------|-----------------------------------------------------------------------------|------------------------------------------------------------------------|
+| `* * *`  | user             | add new contacts                                                            | save the contact information of people                                 |
+| `* * *`  | user             | delete a contact                                                            | free up space in my app                                                |
+| `* * *`  | user             | view all contact                                                            | retrieve contact information                                           |
+| `* * *`  | user             | save all contact                                                            | retain all information for when i reopen the app                       |
+| `* * *`  | sales rep        | have a low query time                                                       | avoid wasting much time querying my desired contact                    |
+| `* *`    | user             | find a person by name                                                       | locate details of persons without having to go through the entire list |
+| `* *`    | user             | search through my contacts                                                  | find a specific person                                                 |
+| `* *`    | new user         | see usage instructions                                                      | know how to use the app                                                |
+| `* *`    | user             | edit contact                                                                | update contact with new information                                    |
+| `* *`    | user             | sort contact by name                                                        | see whose contact I have saved                                         |
+| `* *`    | user             | archive contact                                                             | hide less frequently used contacts without deleting them               |
+| `* *`    | user             | be alerted when a contact already exist                                     | avoid accidentally creating a duplicate                                |
```

Comment:
I dont think this is a valid use case? The user is not trying to achieve any action.